src/app.js

```diff
@@ -1,21 +1,22 @@
 'use strict';
 
-/**
- * Implement sum function:
- *
- * Function takes 2 numbers and returns their sum
- *
- * sum(1, 2) === 3
- * sum(1, 11) === 12
- *
- * @param {number} a
- * @param {number} b
- *
- * @return {number}
- */
-function sum(a, b) {
-  // write code here
-  return a + b;
-}
+const http = require('http');
 
-module.exports = sum;
+const PORT = process.env.PORT || 3000;
+
+const server = http.createServer((req, res) => {
+  const url = new URL(req.url, `http://${req.headers.host}`);
+  const pathname = url.pathname.split('/').filter(el => el !== '');
+  const params = Object.fromEntries(url.searchParams.entries());
+
+  res.writeHead(200, {
+    'Content-Type': 'aplication/html',
```

Comment:
Content-Type is set as 'aplication/html'. It should be 'application/json' since you are returning a JSON response.